src/scripts/events/domEvents.js

```diff
@@ -52,7 +52,10 @@ const domEvents = (uid) => {
         deleteAuthorsBooks(firebaseKey).then((authorsArray) => showAuthors(authorsArray));
       }
     }
-
+    if (e.target.id.includes('fav-author-btn')) {
+      const [, firebaseKey] = e.target.id.split('--');
+      console.warn(firebaseKey);
```

Comment:
We want to make sure not to push any console.warn or logs. They should all be removed